### DISCUSSION TITLE
Supporting Telegram icon in the Profile Socials

### DISCUSSION THREAD

**Discussion by BaseMax** [community/community#48353] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

General

### Body

Telegram is so popular messenger in the world and it's growing so fast. It will be great if GitHub supports and handle the Telegram icon if someone enters Telegram links in the bio.

![image](https://user-images.githubusercontent.com/2658040/220828908-b1444f48-df4e-4771-9010-68d8086ed5a8.png)

Best,
M.

**Comment by asadbekkuz** [comment]
I actually agree with you. Telegram messanger is popular in asia's country, therefore Github has not added this icon.

**Reply by BoiDontCry** [reply]
bro why it is popular in asia?

**Reply by RobyRew** [reply]
> bro why it is popular in asia?

I guess same as here in Europe, to avoid censorship and dictatorship regimes.

Lately absolutism is widely accepted on almost all countries. But Telegram is still neutral and embraces free speech.

**Reply by BoiDontCry** [reply]
i think u r right 
have a good day

**Comment by RobyRew** [comment]
here in Europe is also used a lot, please add it

**Comment by MrMadaraUchiha** [comment]
Poderiam adicionar no Brasil também né. (They could add it in Brazil too, right?)

**Comment by sebastian-nowak** [comment]
I'm surprised that it's missing. Telegram is becoming increasingly more popular in Europe too, I'm using it myself. It definitely should be supported.

**Comment by Deepsonar2004** [comment]
yes

**Comment by ShuhaibNC** [comment]
_**waiting for it**_

**Comment by MohamadeRahbar** [comment]
Yeap, it's true
in Asia its most popular and useful instead of discord, viber and others
so it's possible that GitHub ignored it! 
but we are looking for it :)

**Comment by aiktb** [comment]
Even BlueSky has received support, has GitHub really noticed this issue after a year? How can we let them know?

**Reply by drobit** [reply]
Hello, any updates ?

**Comment by ApostolFet** [comment]
waiting for it

**Comment by Rooterts** [comment]
Me too wait for it :(

**Comment by pjpscriv** [comment]
@BaseMax as a workaround I created a Chrome extension to do this while we wait for GitHub to get to it - https://chromewebstore.google.com/detail/github-social-icons/ledidlccnpfooopagmkgihgibfbnbbnp

![image](https://github.com/user-attachments/assets/cd8ae7cb-1315-41b4-bf97-9f5365d33d64)

**Reply by BaseMax** [reply]
Cool and good.

**Comment by Uffynn** [comment]
Intersting that is has been so long and still nothing? In the meanwhile telegram became the worlds mostly used app between devs and if not in the top3 in the world, overall. And we still didn't get our icon??? 👀
